src/components/study-groups/StudyGroup.js

```diff
@@ -120,18 +147,29 @@ class StudyGroup extends Component {
 }
 
 const styles = theme => ({
+    title: {
+        fontSize: '2rem',
+        [theme.breakpoints.up('md')]: {
+            fontSize: '3rem'
+        }
+    },
+    headerContent: {
+        width: `calc(100% - ${theme.spacing.unit*2}px - 72px)`,
+    },
+    headerAction: {
+        width: 'auto',
+    },
     moment: {
         display: 'block',
         color: theme.palette.text.hint,
         margin: theme.spacing.unit,
     },
-    status: {
-        margin: theme.spacing.unit,
-        color: theme.palette.common.white,
-    },
     markdown: {
         color: theme.palette.text.primary,
     },
-});
+    contactInfo:{
+        padding: '16px 24px 16px 24px' 
+    },
```

Comment:
Qua dovresti prendere le mie gio, per farle complete anche in mobile
```
contactInfo: {
        display: 'inline-flex',
        [theme.breakpoints.down('xs')]: {
            display: 'block',
            paddingLeft: '12px',
        }
    }
```

Reply:
fatto ma ho messo anche un breakpoint up('md') che quando è grande lo rimette a 24